### PR TITLE
fix MaxBlockSystemFee

### PR DESCRIPTION
**Pull request by superboyiii**
Close https://github.com/neo-project/neo-modules/issues/702

### CODE REVIEW

**Review by Liaojinghui — DISMISSED** [review]

File: src/DBFTPlugin/DBFTPlugin/config.json [hunk 1]
```diff
@@ -5,6 +5,6 @@
     "AutoStart": false,
     "Network": 860833102,
     "MaxBlockSize": 2097152,
-    "MaxBlockSystemFee": 100000000000
+    "MaxBlockSystemFee": 300000000000
```
Comment (roman-khimov) [inline src/DBFTPlugin/DBFTPlugin/config.json:8]: 1500 or 2000 GAS? 3K GAS is still quite a lot of CPU time.

Comment (superboyiii) [reply src/DBFTPlugin/DBFTPlugin/config.json]: > 1500 or 2000 GAS? 3K GAS is still quite a lot of CPU time.

Done.

**Review by roman-khimov — DISMISSED** [review]

**Review by Liaojinghui — DISMISSED** [review]

**Comment by erikzhang** [comment]
https://github.com/neo-project/neo-modules/blob/32aacc468ad43600817daabbec834e715017d962/src/DBFTPlugin/Settings.cs#L21

This line should be updated too.

**Comment by superboyiii** [comment]
> 900000000000L

Done

**Review by erikzhang — APPROVED** [review]

**Pull request merged**